cif_core.dic

```diff
@@ -19286,6 +19286,593 @@ save_ATOM
+;
+    _atom_analytical.analyte_mass_percent = dREL to calculate wt% from:
+                                         _atom_analytical.chemical_species and
+                                _atom_analytical.chemical_species_mass_percent
+;
+
+save_
+
+save_atom_analytical.analyte_mass_percent_su
+
+    _definition.id                '_atom_analytical.analyte_mass_percent_su'
+    _definition.update            2022-11-19
+    _description.text
+;
+    Standard uncertainty of _atom_analytical.analyte_mass_percent.
+;
+    _name.category_id             atom_analytical
+    _name.object_id               analyte_mass_percent_su
+    _name.linked_item_id          '_atom_analytical.analyte_mass_percent'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_atom_analytical.chemical_species
+
+    _definition.id                '_atom_analytical.chemical_species'
+    _definition.update            2022-11-19
+    _description.text
+;
+    Chemical formula of the species for which the corresponding
+    _atom_analytical.chemical_species_mass_percent refers.
+
+    The following rules apply in the construction of the formula:
+
+    1. Only recognized element symbols may be used.
+
+    2. The first element corresponds to the analyte. The remaining
+       elements should be given in alphabetic order by symbol.
+
+    3. Each element symbol is followed by a 'count' number. A count of
+       '1' may be omitted.
+
+    4. A space or parenthesis must separate each cluster of (element
+       symbol + count). A formula cannot begin with a paranthesis.
+
+    5. Where a group of elements is enclosed in parentheses, the
+       multiplier for the group must follow the closing parentheses.
+       That is, all element and group multipliers are assumed to be
+       printed as subscripted numbers.
+
+;
+    _name.category_id             atom_analytical
+    _name.object_id               chemical_species
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+
+    loop_
+      _description_example.case
+         'Fe2 O3'
+         'Li'
+         'Si O2'
+
+    _method.purpose               Evaluation
+    _method.expression
+;
+    _atom_analytical.analyte = dREL to extract analyte symbol
+;
+
+save_
+
+save_atom_analytical.chemical_species_mass_percent
+
+    _definition.id
+        '_atom_analytical.chemical_species_mass_percent'
+    _definition.update            2022-11-19
+    _description.text
+;
+    Mass percentage of the chemical species given in
+    _atom_analytical.chemical_species as derived from elemental analysis.
+
+    This is most often used in elemental compositions determined by XRF,
+    where elements are reported as equivalent mass percentages of their
+    relevant oxide.
+
+    eg: Al is reported as Al2O3, P is reported as P2O5.
+
```

Comment:
The trailing empty line is probably not needed.

Reply:
done